_includes/v3.3/release-notes/v3.3.0-release-notes.md

```diff
@@ -0,0 +1,45 @@
+22 October 2018
+
+#### In-cluster route reflector
+
+With Calico v3.3, you can enable the calico/node container to behave as a route reflector on particular nodes in the cluster. calico/node
+in route reflector mode now fully supports route reflector clustering, route reflector peering with external routers, and the
+full set of BGP configuration options supported by Calico. This functionality replaces the calico/routereflector container, which
+is now deprecated.
+
+#### BGP peer configuration using labels
+
+In Calico v3.3, you can now use node labels and selectors to define BGP peering relationships between calico/node instances. This allows
+for easier management of BGP peer configuration in a way that scales automatically with your cluster. Combined with the in-cluster
+route reflector feature, this provides a powerful model for defining how routes are distributed within the cluster.
+
+#### IPAM enhancements
+
+When using Calico IPAM, you can now configure the block size allocated to each host for each IP Pool. The default block size of /26 for
+IPv4 provides blocks of 64 addresses. For details refer to the Project Calico IP Pool documentation. Additionally, cluster administrators
```

Comment:
Is it possible that you might want to hyperlink them to the section of interest?